Assignment1/Assignment1_QNo9/SolAssignmentNo1_QNo9.rb

```
@@ -0,0 +1,14 @@
+# => 9. How to convert 'ruby' to :ruby ?
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def cvt_to_sym
+  # => Repeateadly asking users to give correct input strings
+  puts "Enter string"
+  str = gets.chomp
+  puts "The required symbolic name is #{str.to_hash.value}"
```

@HrisikeshSinghaChowdhury How this code is working for you? it should throw error undefined method `to_hash' for String

---

I mentioned this question is undone